### PR TITLE
Move graphics from TournamentGameBase to TournamentGame

### PR DESCRIPTION
**Pull request by MiraiSubject**
Right now `TournamentGameBase` also adds all the user interface elements on startup. In this PR I have moved all the user interface related code inside `TournamentGameBase` to `TournamentGame` to match the relationship that `OsuGameBase` and `OsuGame` have.

Afaik `OsuGameBase` does not add any drawables to the screen, so this PR is supposed to achieve the same goal, but for the tournament side. 

This allows for headless tests to be created for the Tournament Client as well.

### CODE REVIEW

**Comment by bdach** [comment]
Did you have a particular end goal when making this change? I'm not sure this is worth doing purely for the sake of it.

**Comment by MiraiSubject** [comment]
I noticed that I wasn't able to headlessly test the tournament asset migration that I've been implementing in: https://github.com/MiraiSubject/osu/tree/tourney-asset-refactor

In order to prevent it from turning into a really big pull request that contains this change as well, I thought it would be better to split it up.

File: osu.Game.Tournament.Tests/TournamentTestBrowser.cs [hunk 1]
```diff
@@ -19,6 +19,9 @@ protected override void LoadComplete()
                 Depth = 10
             }, AddInternal);
 
+            MenuCursorContainer.Cursor.AlwaysPresent = true;
```
Comment (peppy) [inline osu.Game.Tournament.Tests/TournamentTestBrowser.cs:22]: would it not make sense to leave this in the base implementation if it's done in both places?

Comment (MiraiSubject) [reply osu.Game.Tournament.Tests/TournamentTestBrowser.cs]: Right, I didn't consider that. I moved back to the base implementation

**Review by bdach — APPROVED** [review]
seems ok 👍

**Review by peppy — APPROVED** [review]

**Pull request merged**